src/index.html

```diff
@@ -1,12 +1,660 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@100;400;500;600&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Catalog</h1>
+    <header class="header header--container">
+      <a href="#logo" class="logo">
+        <img src="./images/logo.png" alt="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <a href="#apple" class="nav__link is-active">
+            Apple
+          </a>
+          </li>
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#phone" class="nav__link">
+              Смартфоны и телефоны
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#noutbuk"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#gadget" class="nav__link">
+              Гаджеты
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#tablet" class="nav__link">
+              Планшеты
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#foto" class="nav__link">
+              ФОТО
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#video" class="nav__link">
+              ВИДЕО
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <section class="card">
+        <section class="card__item card__item--container" data-qa="card">
```

Comment:
```suggestion
    <main class="main">
      <section class="main__cards-grid">
        <section class="main__card card card--container" data-qa="card">
```

Let's follow the structure:
`main` (it's your content root block)
|_ `main__cards-grid` (it's an element-wrapper for your cards inside `main`-block)
    |_ `main__card card` (here you'll have card wrapper `main__card` inside `main` and `card`-block as container for card)
    |    |_ `card__foto`
    |    |_ `card__h2` ...
   ...
    |_ .main__card card (all your cards share the same principle)

Look at [#8 here](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md) and check [BEM-mistakes here](https://mate-academy.github.io/fe-program/css/typical-bem-mistakes).

Reply:
А тут, мені здається, що суперечливе зауваження. Бо main - це такий глобальний тег, який включає і інші блоки. А < section  class =" card " > це саме блок картки. І основна суть BEM , щоб можна було взяти блок "card" і перенести його наприклад в якусь бічну панель. І не змінювати при цьому назву класу блоку.

Reply:
@KorniychukLesya Of course, your `card` is separate BEM-block. But now it's a part of `main` block. When we add `main__card` as element style we can set rules of **positioning** of card **inside** `main`. Card itself doesn't know were it will be placed. `card`-block says us **how it looks** but not **were it locates**. So my proposal is to add such wrapper from `main` block. What do you think?

Reply:
Я б не робила, мабуть, обгортку. У мене вже є "card__item --container" в якому я задала зовнішні відступи, максимум, ще б  в цей контейнер перенесла ширину картки. Мені так здається.

Reply:
@KorniychukLesya `card__item--container` is a modifier. Modifier describes **state** of an element. In your case your `card` isn't in a modified state - it's only **positioned** inside your `main__grid` as `main__card` (<-- this class describes positioning as container).
So, the right way is to set margins and sizes in `main__card`, and leave `card` in default state (remove `card--container`).